projects/experimental/directives/button-group-card/button-group-card.style.less

```diff
@@ -0,0 +1,52 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButtonGroupCard] {
+    display: flex;
+    min-width: 21.4375rem;
+    border-radius: 1.5rem;
+
+    &[data-surface='dark'] > [tuiButton][tuiAppearance] {
+        color: #fff;
+
+        &:hover {
+            background: rgba(256, 256, 256, 0.13);
```

Comment:
Did you check with designers if we need a hover effect?

Reply:
drop